Update/wata_003.txt

```diff
@@ -6213,7 +6214,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#956f6e>圭一</color>", NULL, "<color=#956f6e>Keiichi</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 1, "ps3/s19/01/hr_kei15020", 128, TRUE);
 	OutputLine(NULL, "「……詩音、",
-		   NULL, "\"...Shion,", Line_Continue);
+		   NULL, "\"...Shion", Line_Continue);
```

Comment:
The comma should be here.  I think any case where you have a partial sentence like this it makes sense to leave any comma in place.  I think i've seen one other case like this so far so please review to check other cases.  It's especially fitting here because it was going to say before the name change "...Shion, when she heard I was so hungry..." which would be grammatically correct.

Reply:
Thing is, a comma becomes redundant with an ellipsis, so I don't think having the comma is necessary since the ellipsis already takes it's role

Reply:
I saw your comment on that before and I would agree with a single-line sentence but when the sentence spans multiple script lines I think that's where the case for leaving the comma in is strong.  @enumag what do you think?

Reply:
Also @Norgus since you're a native speaker too right

Reply:
I also think we should keep the commas.

Reply:
I see the ellipsis just as something that signifies a pause when talking. Comma on the other hand is a grammer thing. Commas and ellipsis can appear independently on each other or together because they have a different meaning in my opinion.

(Note: I'm not native english speaker. Just saying what I think.)

Reply:
I'm afraid I don't really know what's technically correct and don't have particularly strong feelings on the matter.